### PR TITLE
Bumped Gradle and enabled Build Scans

### LINKED ISSUE #544: improve stubbing warnings formatting
Follow up after #384, let's get the stubbing warnings look good in case there are many instances to report (text padding).

### LINKED ISSUE #564: Publish Mockito build results to Gradle Build Scans
Let's start publishing Mockito build results to Gradle Build Scans. This way we have extra intel about Gradle builds. This also helps friends at Gradle as they will have more projects on board!

### PR DESCRIPTION
**Pull request by szczepiq**
- Fixes #564 
- Bumped Gradle to latest 2.* (regenerated Gradle wrapper)
- Added build scans support
- Removed TODO I decided to not work on (fixes #544)

### CODE REVIEW

File: gradlew.bat [hunk 1]
```diff
@@ -46,7 +46,7 @@ echo location of your Java installation.
 goto fail
 
 :init
-@rem Get command-line arguments, handling Windowz variants
+@rem Get command-line arguments, handling Windows variants
```
Comment (TimvdLippe) [inline gradlew.bat]: :joy:

**Comment by TimvdLippe** [comment]
This is an example build scan of Gradle: https://scans.gradle.com/s/3xeftfslzye3u

I had to enter my e-mail address for this. Does every Mockito developer have to do this? Also could we maybe automatically retrieve this build scan link and put it in the status checks of GitHub?

Other than that, it looks really useful. Apparently 1 test took 6 seconds :O

**Comment by szczepiq** [comment]
>I had to enter my e-mail address for this. Does every Mockito developer have to do this? Also could we maybe automatically retrieve this build scan link and put it in the status checks of GitHub?

Possibly. Not a high priority to me atm ;)

**Comment by breskeby** [comment]
@szczepiq could you sent me a link with a sample build AND your avatar on it? That would be great to have on the gradle.com website for our open source projects section.

**Comment by bric3** [comment]
@breskeby It's better to not affiliate this to a single person but rather the project !

**Comment by bric3** [comment]
@TimvdLippe most long test are related to concurrency

**Comment by TimvdLippe** [comment]
Small follow up: our build is now featured on the homepage of gradle.com with the following image: 
image

**Pull request merged**